infrastructure/docker-images/build.sh

```diff
@@ -7,6 +7,14 @@ if [ -z "${node_version}" ]; then
     exit 1
 fi
 
+
+go_version=$(<${repo_root}/.go-version)
```

Comment:
we build two images with one command? If one of the `docker build` fails - I presume we still get the second image? But then if we have to re-run the command the second image gets built again?

Reply:
This is the third one, actually.  It probably makes sense to split them into separate scripts as, yes, it will rebuild the other images and end up with multiple different images with the same tag.  I'll fix that up in another push and then land.  Thanks!

These builds don't take too long, as they're mostly just based on existing images.